draft-ietf-quic-transport.md

```diff
@@ -6636,14 +6636,17 @@ restricting the length of time an endpoint is allowed to stay connected.
 
 ## Stream Fragmentation and Reassembly Attacks
 
-An adversarial sender might intentionally send fragments of stream data in an
-attempt to cause disproportionate receive buffer memory commitment and/or
-creation of a large and inefficient data structure.
+An adversarial sender might intentionally omit to send portions of the stream
+data, causing the receiver to commit resources for the omitted data. This could
+cause a disproportionate receive buffer memory commitment and/or the creation of
+a large and inefficient data structure at the receiver.
```

Comment:
```suggestion
An adversarial sender might intentionally not send portions of the stream data,
causing the receiver to commit resources for the unsent data. This could
cause a disproportionate receive buffer memory commitment and/or the creation of
a large and inefficient data structure at the receiver.
```

Reply:
That suggestion works for me, though I think that omit is a good word fit.